packages/blocks/src/api/raw-handling/utils.js

```diff
@@ -18,7 +18,7 @@ const phrasingContentSchema = {
 	em: {},
 	del: {},
 	ins: {},
-	a: { attributes: [ 'href' ] },
+	a: { attributes: [ 'href', 'target' ] },
```

Comment:
We don't have the UI to add other kinds of targets (aside `_blank`). We also add `rel="noreferrer noopener"` to these links. The question is: Should we allow only `target="_blank"` and normalize the "rel" attribute in that case? or it's fine to just pick the target and keep the link as is?

Reply:
Those are good questions. Perhaps we should perform some normalization, as there is an expectation — given the precedents when pasting from word processors, etc. — that Gutenberg generally cleans up markup. I was looking at the raw-handling pipeline and am guessing that an acceptable place to normalize `target` and `rel` would be `phrasing-content-reducer`, next to where special rules are applied to `b` and `i` tags: https://github.com/WordPress/gutenberg/blob/09db90456689a4e83d045bf1fe8ea1614c156ca5/packages/blocks/src/api/raw-handling/phrasing-content-reducer.js#L24-L28

Reply:
Good idea, @mcsf, I added normalisation here.